.neuro/live.yml

```diff
@@ -155,7 +155,11 @@ jobs:
         --nmber_of_imgs ${{ params.extend_dataset_by }} --skip_annotation_update
 
       # Push dataset
-      pachctl put file -o -r ${PACHY_REPO}@master:data/Images/ -f ${DATA_PATH}/ | tee
+      pachctl put file -r ${PACHY_REPO}@master:data/Images/ -f ${DATA_PATH}/ | tee
+      mkdir -p /tmp/ls-completions
+      touch /tmp/ls-completions/test
+      pachctl put file -r ${PACHY_REPO}@master:/ls-completions/ -f /tmp/ls-completions | tee
```

Comment:
redundant?
if we need it to perform checkout at LabelStudio job - let's better do it there.

To work-around cases, when the completions does not exist yet, maybe we could do smth like
`pachctl get file ${PACHY_REPO}@master:/ls-completions/ -r -o ${COMPLETIONS} | tee || echo "No completions yet`

Reply:
done